src/linter/block.go

```diff
@@ -495,7 +495,7 @@ func (b *blockWalker) walkComments(n ir.Node, c freefloating.String) {
 
 		types, warning := typesFromPHPDoc(&b.r.ctx, p.Type)
 		if warning != "" {
-			b.r.Report(n, LevelInformation, "phpdocType", "%s on line %d", warning, p.Line())
+			b.r.Report(n, LevelWarning, "phpdocType", "%s on line %d", warning, p.Line())
```

Comment:
I think phpdoc mistake is a notice, not a warning. It's not even a part of the executable PHP code.